conjure-java-client-verifier/src/test/java/com/palantir/conjure/java/compliance/VerificationClients.java

```diff
@@ -18,16 +18,30 @@
 
 import com.palantir.conjure.java.api.config.service.UserAgent;
 import com.palantir.conjure.java.client.jaxrs.JaxRsClient;
+import com.palantir.conjure.java.dialogue.serde.DefaultConjureRuntime;
 import com.palantir.conjure.java.okhttp.HostMetricsRegistry;
 import com.palantir.conjure.verification.server.AutoDeserializeConfirmService;
 import com.palantir.conjure.verification.server.AutoDeserializeService;
+import com.palantir.conjure.verification.server.BlockingAutoDeserializeService;
+import com.palantir.conjure.verification.server.BlockingSingleHeaderService;
+import com.palantir.conjure.verification.server.BlockingSinglePathParamService;
+import com.palantir.conjure.verification.server.BlockingSingleQueryParamService;
+import com.palantir.conjure.verification.server.DialogueAutoDeserializeService;
+import com.palantir.conjure.verification.server.DialogueSingleHeaderService;
+import com.palantir.conjure.verification.server.DialogueSinglePathParamService;
+import com.palantir.conjure.verification.server.DialogueSingleQueryParamService;
 import com.palantir.conjure.verification.server.SingleHeaderService;
 import com.palantir.conjure.verification.server.SinglePathParamService;
 import com.palantir.conjure.verification.server.SingleQueryParamService;
+import com.palantir.dialogue.JavaChannels;
+import com.palantir.tritium.metrics.registry.DefaultTaggedMetricRegistry;
 
 public final class VerificationClients {
     private VerificationClients() {}
 
+    private static final DefaultConjureRuntime DEFAULT_CONJURE_RUNTIME =
+            DefaultConjureRuntime.builder().build();
```

Comment:
I wonder if at some point we could dedupe the undertow & dialogue serializer registries?

Reply:
I don't think we should, they cover similar concepts for separate uses so we would lose flexibility. I don't think they're large or complex enough for that to be worthwhile.